_content/monographs/wonhyo-selected-works_muller-park-vermeersch.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Wonhyo Selected Works"
+authors:
+  - "Wonhyo"
+editor: "A. Charles Muller"
+translator: A. Charles Muller and Jin Y. Park, and Sem Vermeersch"
+external_url: "http://www.acmuller.net/kor-bud/01_wonhyo.pdf"
```

Comment:
Sorry. No http pdfs allowed. Find a https link or use the wayback machine as a proxy: https://web.archive.org/web/20250505111157if_/http://www.acmuller.net/kor-bud/01_wonhyo.pdf